.github/workflows/release-main.yml

```diff
@@ -90,3 +90,70 @@ jobs:
           commit-message: "feat: bump kib to ${{ needs.release-please.outputs.release_tag_name }}"
           path: cluster-api-provider-preprovisioned
           title: "feat: bump kib to ${{ needs.release-please.outputs.release_tag_name }}"
+
+  sign-binary:
+    runs-on: macos-latest
+    needs: release-please
+    steps:
+      - if: ${{ needs.release-please.outputs.release_created }}
+        uses: dsaltares/fetch-gh-release-asset@1.0.0
+        with:
+          version: "tags/${{ needs.release-plese.outputs.release_tag_name }}"
+          file: "konvoy-image-bundle-${{ needs.release-plese.outputs.release_tag_name }}_darwin_amd64.tar.gz"
+
+      - if: ${{ needs.release-please.outputs.release_created }}
+        name: Create directory to extract the pulled file into
+        run: mkdir "konvoy-image-bundle_darwin_amd64"
+
+      - if: ${{ needs.release-please.outputs.release_created }}
+        name: Extract the pulled file
+        run: tar -xvzf "konvoy-image-bundle-${{ needs.release-plese.outputs.release_tag_name }}_darwin_amd64.tar.gz" -C "konvoy-image-bundle_darwin_amd64"
+
+      - if: ${{ needs.release-please.outputs.release_created }}
+        name: Remove original downloaded file
+        run: rm "konvoy-image-bundle-${{ needs.release-plese.outputs.release_tag_name }}_darwin_amd64.tar.gz"
+
+
+      - if: ${{ needs.release-please.outputs.release_created }}
+        name: Fetch current main directory
+        uses: actions/checkout@v3
+        with:
+          ref: main
+          fetch-depth: 0
+
+      - if: ${{ needs.release-please.outputs.release_created }}
```

Comment:
Should cleanup the certs in the end.
https://github.com/Apple-Actions/import-codesign-certs/issues/8#issuecomment-1011294106

Reply:
Ah, yes I commented that earlier.